backend/src/main/java/de/neuefische/backend/service/AppointmentsService.java

```diff
@@ -0,0 +1,41 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.BackendApplication;
+import de.neuefische.backend.model.AppointmentCreationDTO;
+import de.neuefische.backend.model.Appointments;
+import de.neuefische.backend.model.UserMongo;
+import de.neuefische.backend.repo.AppointmentsRepository;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class AppointmentsService {
+
+    private static final Log LOG = LogFactory.getLog(BackendApplication.class);
+    private final AppointmentsRepository appointmentsRepository;
+
+    public AppointmentsService(AppointmentsRepository appointmentsRepository) {
+        this.appointmentsRepository = appointmentsRepository;
+    }
+
+    public List<Appointments> getAppointmentsOfUser(UserMongo userMongo){
+        LOG.info("Loading " + userMongo.getUsername() + "'s appointments");
+        return appointmentsRepository.findAllByUsername(userMongo.getUsername());
+    }
+
+    public Appointments createAppointment(@Validated AppointmentCreationDTO data, String username){
```

Comment:
DTOs define your api and the service does not have to know about your api. Passing the entries from the dto as individual parameters leads to cleaner separation between your layers

Reply:
I guess you're right.
I copied my 'createAppointment' from my 'createBagPlace' which i wrote similar with Florians' help.

Reply:
Don't worry that's not a big thing and it is perfectly okay to have different opinions on this. Just wanted to point it out.